subjects/get-json/README.md

```diff
@@ -5,15 +5,15 @@
 In this exercise, we will focus on building complex async flows with promises.
 
 Create a `getJSON` function that takes 2 parameters:
-- `path`, that will be the url called by your function
-- `params` *optional*, that will be the search parameters appended to your url
+- `path`, string that will be the url called by your function
+- `params` *optional*, object that will be the search parameters appended to your url
 
 `getJSON` must construct a valid url with the `path` and stringified `params`
 and call `fetch` with it.
-If the response is not ok, your function must throw an error using
-the response status message.
+If the response is not ok (i.e. response status is not 2xx), your function must throw an error using
+the HTTP status code message.
```

Comment:
I'm good with adding HTTP, but it's not the code if it's the message, it can't be both, a status as a code and a message not a code message.

Maybe specify the request status message ?